cobalt/testing/failing_tests_filter.json

```diff
@@ -0,0 +1,106 @@
+{
+  "filters": [
```

Comment:
Also if you rename this key from "filters" to "excluded_tests" perhaps it makes more obvious sense, and you can omit the `-` sign from the filter names and add that in your jquery expression. Would make it less visually noisy and easier to comprehend what's going on in the file

Reply:
Thanks, done.